runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNet.java

```diff
@@ -37,10 +40,23 @@
 	private final int objectId;
 	private final Varbits statusVarbit;
 	private final Varbits countVarbit;
+	private final Set<WorldPoint> adjacentTiles;
 
 	private GameObject net;
 	private DriftNetStatus status;
 	private int count;
+	@Setter
+	private DriftNetStatus prevTickStatus;
+
+	boolean isNotAcceptingFish()
+	{
+		return status != DriftNetStatus.CATCH && status != DriftNetStatus.SET;
+	}
+
+	boolean wasNotAcceptingFish()
+	{
+		return prevTickStatus != DriftNetStatus.CATCH && prevTickStatus != DriftNetStatus.SET;
+	}
```

Comment:
Since we don't actually care if we were or are accepting fish (`.filter(net -> net.isNotAcceptingFish() || net.wasNotAcceptingFish())`) maybe we should just combine this into one `notAcceptingFish` and put the comment here?